### DISCUSSION TITLE
Email administrators when a purchase requires approval

### DISCUSSION THREAD

**Discussion by tgeller** [pretix/pretix#4389] [Ideas] [answered]
1. As an administrator, I created a product that requires admin approval.
2. As a user (with a different email address), I purchased this product.
3. As a user, I received a confirmation email that said admin approval is required -- great!
4. As an administrator, I got no email notice that the user's order was waiting for my approval. I asked other Pretix admins and they said the admin has to just keep checking the web admin interface.

Request: Send an admin email whenever an order requires admin action.

I imagine this is relatively easy. Good luck!

**Comment by raphaelm** [comment, marked as answer]
You can already set this up:
- Click on your name or email in the top-right corner of the backend
- Click "Change notification settings"
- Activate notifications for "New order requires approval"

**Reply by tgeller** [reply]
Thank you! Sorry for missing something so obvious.
